### ISSUE TITLE
Week 3 announcements

### ISSUE THREAD

**Issue by joenash** [campus-experts/spring-2017#20]
## The TLDR this week:

- [Week 3 todo](https://github.com/campus-experts/spring-2017/blob/master/todos/week-3.md)
- [Last weeks video](https://www.dropbox.com/s/onvv9hdqwgptpww/week2.mp4?dl=0), what to expect in week 3 and some discussion about particular community assessments, not relevant to all!
- Submit a [Pull Request](https://help.github.com/articles/creating-a-pull-request/) with your [Community Impact Proposal](https://github.com/campus-experts/spring-2017/blob/master/docs/community-impact-proposal.md)
- Check out and comment on other people's [community assessments](https://github.com/campus-experts/spring-2017/pulls?q=is%3Apr+is%3Aopen+label%3Acommunity-assessment).
- This week we have two guest talks! Git/GitHub training on Tuesday 11th April, and a GraphQL workshop on 12th April. 

## Welcome to Week 3 🚩

Thanks so much for getting your community assessments in. I'll try and make sure they all have comments and feedback very soon. Now it's the week to start reflecting on those assessments and thinking "how do I solve these challenges?". You'll create an Impact Proposal detailing *something* you'll enact in your community: a change, an event, a program, something that addresses the problems in your community and makes a positive impact. 

This week is a bit of a busy week as we have a lot of guest talks! The remaining call slots this week will be for reviewing and discussing your proposals. 

## This week on Slack
Missed a discussion on slack? Not to worry!

- We're building a list of [resources](https://githubeducation.slack.com/archives/C2TQDRQSG/p1491837725965935)! 
- Where do you want discussions to [happen](https://githubeducation.slack.com/archives/C2TQDRQSG/p1491916310177519)?
- The role of clubs in helping members find [jobs](https://githubeducation.slack.com/archives/C2TQDRQSG/p1491523703801967)
- Prizes and hacker [motivation](https://githubeducation.slack.com/archives/C2TQDRQSG/p1491414683562922)
